cli/src/cmd_auth.rs

```diff
@@ -39,6 +40,7 @@ pub struct CmdAuth {
 enum SubCommand {
     Login(CmdAuthLogin),
     Logout(CmdAuthLogout),
+    RenameProfile(CmdAuthRenameProfile),
```

Comment:
I'd suggest

`oxide auth profile rename --from foo --to bar`

In the future I could imagine `oxide auth profile output --json` or something.